tests/BenchmarkDotNet.Tests/Attributes/VerifiedFiles/ParamsAllValuesVerifyTests.BenchmarkShouldProduceSummary_WithAllValuesOfBool.verified.txt

```diff
@@ -7,9 +7,9 @@ Frequency: 2531248 Hz, Resolution: 395.0620 ns, Timer: TSC
   DefaultJob : extra output line
 
 
-    Method | ParamProperty |     Mean |   Error |  StdDev |
----------- |-------------- |---------:|--------:|--------:|
- Benchmark |         False | 102.0 ns | 6.09 ns | 1.58 ns | ^
- Benchmark |          True | 202.0 ns | 6.09 ns | 1.58 ns | ^
+Method    | ParamProperty |     Mean |   Error |  StdDev |
```

Comment:
Can you make all headers uniform justification? All left or all right, independent of the column's justification. It may make sense to use the `TextJustification` property (remove "Column" from the name).

Reply:
I'm a little confused. Correct me if I'm wrong, you want to have 2 styles: "TextJustification" and "NumericColumnJustification". "TextJustification" is used for styling header texts as well as text columns. "NumericColumnJustification" is used for styling numeric columns.

Reply:
I would like the headers to have uniform justification. The way I see it, there are 3 options for this.

1. Always left-justify headers (or right-justify), regardless of settings.
2. Use the same settings as text cells (think of the settings for each type of cell, rather than column).
3. Add a separate setting for header justification.

@AndreyAkinshin Do you have an opinion on this?

Reply:
> I would like the headers to have uniform justification. The way I see it, there are 3 options for this.
> 
> 1. Always left-justify headers (or right-justify), regardless of settings.
> 2. Use the same settings as text cells (think of the settings for each type of cell, rather than column).
> 3. Add a separate setting for header justification.
> 
> @AndreyAkinshin Do you have an opinion on this?

I implemented the first option. Headers are now always justified to left.